cmake/DARTFindassimp.cmake

```
@@ -12,7 +12,7 @@ find_package(assimp REQUIRED MODULE)
 # doesn't allow different major number while DART is compatible any version
 # greater than or equal to 3.2.
 if(ASSIMP_VERSION VERSION_LESS 3.2)
```

Using the config file seems not working as it is (https://dev.azure.com/dartsim/dart/_build/results?buildId=1636). Let's check the version only when the variable is valid to workaround.

```suggestion
if(ASSIMP_VERSION AND ASSIMP_VERSION VERSION_LESS 3.2)
```

---

the whitespace looked funny in that suggestion, but I've committed it in 4567f84

---

Yeah, it seems the suggestion feature is glitchy with indentation sometimes, but it's actually fine when it's applied. Anyways, the change looks good to me!